bubblesub/spell_check/enchant.py

```diff
@@ -16,6 +16,8 @@
 
 """Spell checker based on the pyenchant library."""
 
+# pylint: disable=no-member
```

Comment:
Why is this put also in the constructor of the class below? Doesn't declaring this at the module level suffice?

Reply:
My bad, I will remove it from the constructor of the class, thanks for catching it :)